stbt/__init__.py

```diff
@@ -25,6 +25,7 @@
 from collections import deque, namedtuple
 from contextlib import contextmanager
 from distutils.version import LooseVersion
+import traceback
```

Comment:
Can we keep the PEP8 import order here (alphabetical, "import blahs" before "from blah import blahs").